.github/workflows/merge-python-requirements-upgrade-prs.yml

```diff
@@ -0,0 +1,87 @@
+name: merge-upgrade-prs
+
+on:
+  schedule:
+    - cron: "0 7 * * *"
+jobs:
+  get-list:
+    runs-on: ubuntu-20.04
+    outputs:
+      urls: ${{ steps.generate-matrix.outputs.urls }}
+    steps:
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+      - name: Checkout tools repo
+        uses: actions/checkout@v3
+        with:
+          repository: openedx/tubular
+          path: tubular
+
+      - name: setup tubular
+        run: |
+          cd tubular
+          pip install -e .
+      - name: get urls of all eligible prs
+        env:
+          GIT_TOKEN: ${{ github.token }}
+        id: generate-matrix
+        run: |
+          echo "urls=$(get_ready_to_merge_prs.py --org openedx | sed '1d;2d;')" >> $GITHUB_OUTPUT
```

Comment:
I made some review comment on the `get_ready_to_merge_prs.py` script: https://github.com/openedx/tubular/commit/b00077e3fe0724ee93d2d69918441d9ae4b1e3dd

Is there a reason to have that script in the tubular repo if it's used from here?

Reply:
We were using a few other scripts from that repo, so just hosted ours there too, otherwise was no specific reason.